core/arch/arm/include/mm/core_mmu.h

```diff
@@ -73,6 +74,14 @@
 #define CORE_MMU_USER_PARAM_SIZE	(1 << CORE_MMU_USER_PARAM_SHIFT)
 #define CORE_MMU_USER_PARAM_MASK	(CORE_MMU_USER_PARAM_SIZE - 1)
 
+#ifndef CFG_TEE_RAM_VA_SIZE
+#define CFG_TEE_RAM_VA_SIZE		CORE_MMU_PGDIR_SIZE
+#endif
+
+#ifndef STACK_ALIGNMENT
+#define STACK_ALIGNMENT   (sizeof(long) * 2)
```

Comment:
It's a side effect of bringing a default value for `CFG_TEE_RAM_VA_SIZE`. As **core_mmu.h** must include **platform_config.h**, we must also prevent the `STACK_ALIGNMENT` from user_ta.c to conflict with the one from platform_config.h, hence providing also a default value in **core_mmu.h** for it.
I will add this description in the commit [core: default define stack alignment and core vmem size](https://github.com/OP-TEE/optee_os/pull/1468/commits/499cd4b2fbde67f54dd0b849ee9941cbaf9c3ed7).